magellan-sample-advanced/src/main/java/com/wealthfront/magellan/sample/advanced/toolbar/ToolbarHelper.kt

```diff
@@ -0,0 +1,38 @@
+package com.wealthfront.magellan.sample.advanced.toolbar
+
+import android.content.Context
+import com.wealthfront.magellan.Navigator
+import com.wealthfront.magellan.navigation.NavigableCompat
+import com.wealthfront.magellan.navigation.NavigableListener
+
+object ToolbarHelper : NavigableListener {
```

Comment:
I think I mentioned this in our conversation, but we should consider changing the API of this example to use a data class instead of multiple methods. That way, we configure all the options at once (and have a convenient map of all possible states for the toolbar)

Reply:
Agreed. This is the way to go. Will probably do this at a later point in time once we finalized the API and offer it through our support library.